### DISCUSSION TITLE
Where can I find the estimated 1y target est for a symbol?

### DISCUSSION THREAD

**Discussion by Oooooor10** [gadicc/yahoo-finance2#738] [Q&A]
Hey everybody!
On yahoo finance, on the main symbol page, you can see the estimated 1 year target price on the bottom right corner.
I couldn't find this value on any of the requests. Am I missing it or its just not being retrieved?
<img width="1059" alt="Screenshot 2024-02-15 at 12 57 13 PM" src="https://github.com/gadicc/node-yahoo-finance2/assets/94364902/5cea8c83-763a-4643-ae3e-0d8bc3c29f68">

**Comment by Shadaeiou** [comment]
Looks like it's [this](https://github.com/gadicc/node-yahoo-finance2/blob/217032f1668ca87791160911dc171df8a39b3e0f/docs/modules/quoteSummary.md?plain=1#L569). Running that for AAPL matches what's on yahoo finance summary page.
